warehouse/query-core/src/main/java/datawave/query/jexl/visitors/TermCountingVisitor.java

```diff
@@ -39,6 +34,8 @@ public Object visit(ASTAndNode node, Object data) {
         if (JexlASTHelper.findRange().isRange(node)) {
             // count each bounded range as 1
             ((MutableInt) data).increment();
+        } else if (QueryPropertyMarkerVisitor.isIvarator(node)) {
+            ((MutableInt) data).increment();
```

Comment:
With your `isIvarator` addition, ranges are already covered, so we don't need the first condition anymore.

```suggestion
        if (QueryPropertyMarkerVisitor.isIvarator(node)) {
            // count everything that ivarates as 1
            ((MutableInt) data).increment();
```